no-computer.md

```diff
@@ -58,12 +71,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this would require a LOT of systems/semantics work, especially in defining all the terms and outcomes of a baseball game. As such, I can't imagine it would be particularly fun to code.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Extremely ambitious. Huge idea.
```

Comment:
Also one MLB would probably buy if you could perfect (_although that is easier said than done_)!